src/bot/commands/rules.command.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Injectable,
+  Logger,
+  UseFilters,
+  UseInterceptors,
+} from '@nestjs/common';
+import {
+  Command,
+  EventParams,
+  Handler,
+  InteractionEvent,
+} from '@discord-nestjs/core';
+import { SlashCommandPipe } from '@discord-nestjs/common';
+import {
+  CommandInteraction,
+  Message,
+  MessagePayload,
+  PermissionFlagsBits,
+} from 'discord.js';
+import { MikroORM, UseRequestContext } from '@mikro-orm/core';
+import { InjectRepository } from '@mikro-orm/nestjs';
+import { EntityRepository } from '@mikro-orm/postgresql';
+
+import { RulesService } from '../rules.service';
+import { TimingLogInterceptor } from '../interceptors/logging.interceptor';
+import { ValidationErrorFilter } from '../filters/validation-error.filter';
+import { CatchallErrorFilter } from '../filters/catchall-error.filter';
+import { GuildEntity } from 'src/entities/guild/guild.entity';
+
+export class CheckScoreDTO {}
+
+@Command({
+  name: 'rules',
+  description: `Rules for Virginity Bot on your server.`,
+  defaultMemberPermissions: PermissionFlagsBits.SendMessages,
+  dmPermission: false,
+})
+@Injectable()
+@UseFilters(ValidationErrorFilter, CatchallErrorFilter)
+@UseInterceptors(TimingLogInterceptor)
+export class RulesCommand {
+  private readonly logger = new Logger(RulesCommand.name);
+
+  constructor(
+    private readonly orm: MikroORM,
+    @InjectRepository(GuildEntity)
+    private readonly guilds: EntityRepository<GuildEntity>,
+    private readonly rulesBoard: RulesService,
```

Comment:
This should be called `rules` or `rules_service`

Reply:
Valid

Reply:
Addressed.